### ISSUE TITLE
t2i training dataset

### ISSUE THREAD

**Issue by LilyDaytoy** [ByteDance-Seed/Adversarial-Flow-Models#2]
Hi, thanks for the great work!
In the paper Continuous Adversarial Flow Models you mentioned that you use public dataset for training Z-image using your method, may I ask about what specific datasets you are using? Thanks!
